### PR TITLE
SEO-176113-JS-Multiple-H1

### PR DESCRIPTION
**Pull request by berylouma**
Hello,

I have worked on the multiple H1

Regards,
Beryl.

### CODE REVIEW

File: js/SplitButton/Easy-customization.md [hunk 1]
```diff
@@ -1,14 +1,14 @@
 ---
 layout: post
-title: Easy-customization
-description: easy customization
+title: Easy-customization | JavaScript Split Button | Syncfusion
```
Comment (Yvone-Atieno) [inline js/SplitButton/Easy-customization.md:3]: Update title in correct format

File: js/TreeGrid/Context-Menu.md [hunk 2]
```diff
@@ -1,14 +1,14 @@
 ---
 layout: post
-title: Context-Menu
-description: context menu
+title: Context-Menu | JavaScript | Syncfusion
```
Comment (Yvone-Atieno) [inline js/TreeGrid/Context-Menu.md:3]: Check title

**Review by Yvone-Atieno — CHANGES_REQUESTED** [review]
Commented

File: js/SplitButton/Easy-customization.md (hunk 1 above)
Comment (berylouma) [reply js/SplitButton/Easy-customization.md]: Updated

File: js/TreeGrid/Context-Menu.md (hunk 2 above)
Comment (berylouma) [reply js/TreeGrid/Context-Menu.md]: Updated

File: js/SplitButton/Easy-customization.md [hunk 3]
```diff
@@ -238,7 +238,7 @@ Configure the styles.
 
 Execute the above code to render the following output.
 
-![](/js/SplitButton/Easy-customization_images/Easy-customization_img2.png) 
+![Button Size in JavaScript Split Button](/js/SplitButton/Easy-customization_images/Easy-customization_img2.png)
```
Comment (Yvone-Atieno) [inline js/SplitButton/Easy-customization.md:241]: Button size in JavaScript Split Button

File: js/SplitButton/Easy-customization.md [hunk 4]
```diff
@@ -426,7 +426,7 @@ Configure the styles.
 
 Execute the above code to render the following output.
 
-![](/js/SplitButton/Easy-customization_images/Easy-customization_img3.png) 
+![Content Type in JavaScript Split Button](/js/SplitButton/Easy-customization_images/Easy-customization_img3.png)
```
Comment (Yvone-Atieno) [inline js/SplitButton/Easy-customization.md:429]: the second word should start in lower case

File: js/SplitButton/Easy-customization.md [hunk 5]
```diff
@@ -667,7 +667,7 @@ Configure the styles.
 
 Execute the above code to render the following output.
 
-![](/js/SplitButton/Easy-customization_images/Easy-customization_img4.png) 
+![Image Possition in JavaScript Split Button](/js/SplitButton/Easy-customization_images/Easy-customization_img4.png)
```
Comment (Yvone-Atieno) [inline js/SplitButton/Easy-customization.md:670]: Check position spelling
Image position in JavaScript Split Button

File: js/SplitButton/Easy-customization.md [hunk 6]
```diff
@@ -857,7 +857,7 @@ Configure the **CSS** styles to apply on buttons.
 
 Execute the above code to render the following output.
 
-![](/js/SplitButton/Easy-customization_images/Easy-customization_img5.png) 
+![Theme Support in JavaScript Split Button](/js/SplitButton/Easy-customization_images/Easy-customization_img5.png)
```
Comment (Yvone-Atieno) [inline js/SplitButton/Easy-customization.md:860]: After the first word, the second word should be in lower case. Check in all places

File: js/TreeGrid/Context-Menu.md [hunk 7]
```diff
@@ -1,14 +1,14 @@
 ---
 layout: post
-title: Context-Menu
-description: context menu
+title: Context-Menu in JavaScript TreeGrid | Syncfusion
```
Comment (Yvone-Atieno) [inline js/TreeGrid/Context-Menu.md:3]: Add Control after controlname

**Review by Yvone-Atieno — CHANGES_REQUESTED** [review]
Commented

File: js/SplitButton/Easy-customization.md [hunk 8]
```diff
@@ -1,14 +1,14 @@
 ---
 layout: post
-title: Easy-customization
-description: easy customization
+title: Easy-customization in JavaScript Split Button | Syncfusion
```
[the remark this answers is not in the document]
Comment (berylouma) [reply js/SplitButton/Easy-customization.md:3]: Done

File: js/SplitButton/Easy-customization.md [hunk 9]
```diff
@@ -54,7 +54,7 @@ In the **HTML** page, add the following button elements to configure **Split But
 
 Execute the above code to render the following output.
 
-![](/js/SplitButton/Easy-customization_images/Easy-customization_img1.png) 
+![Content For Split Button in JavaScript](/js/SplitButton/Easy-customization_images/Easy-customization_img1.png)
```
[the remark this answers is not in the document]
Comment (berylouma) [reply js/SplitButton/Easy-customization.md:57]: Done

File: js/SplitButton/Easy-customization.md (hunk 3 above)
Comment (berylouma) [reply js/SplitButton/Easy-customization.md:241]: Updated

File: js/SplitButton/Easy-customization.md (hunk 4 above)
Comment (berylouma) [reply js/SplitButton/Easy-customization.md:429]: Updated

File: js/SplitButton/Easy-customization.md (hunk 5 above)
Comment (berylouma) [reply js/SplitButton/Easy-customization.md:670]: Corrected

File: js/SplitButton/Easy-customization.md (hunk 6 above)
Comment (berylouma) [reply js/SplitButton/Easy-customization.md:860]: Updated

File: js/TreeGrid/Context-Menu.md (hunk 7 above)
Comment (berylouma) [reply js/TreeGrid/Context-Menu.md:3]: Done

**Review by Yvone-Atieno — APPROVED** [review]
Approved

**Review by Aishwarya-Ganesan — APPROVED** [review]
Fine.

**Pull request merged**